content/en/docs/instrumentation/js/_index.md

```diff
@@ -14,8 +14,32 @@ weight: 20
 {{% alert title="Warning" color="warning" %}}
 {{% _param notes.browser-instrumentation %}} {{% /alert %}}
 
-## Further Reading
+## Version Support
 
-- [OpenTelemetry for JavaScript on GitHub](https://github.com/open-telemetry/opentelemetry-js)
-- [Getting Started](getting-started/)
-- [SDK and API Reference](https://open-telemetry.github.io/opentelemetry-js)
+OpenTelemetry JavaScript supports all active or maintenance LTS versions of
+Node.js. Previous versions of Node.js may work, but are not tested by
+OpenTelemetry.
+
+OpenTelemetry JavaScript has no official supported list of browsers. It is aimed
+to work on currently supported versions of major browsers.
+
+For more details on runtime support see
+[ths overview](https://github.com/open-telemetry/opentelemetry-js#supported-runtimes).
+
+## Repositories
+
+OpenTelemetry JavaScript consists of the following repositories:
+
+- [opentelemetry-js](https://github.com/open-telemetry/opentelemetry-js), core
+  repository containing the core distribution API and SDK.
+- [opentelemetry-js-contrib](https://github.com/open-telemetry/opentelemetry-js-contrib),
+  contributions that are not part of the core distribution of the API and SDK.
```

Comment:
Probably worth mentioning that it's primarily instrumentations that are maintained by opentelemetry.

Reply:
Is this wording better? https://opentelemetry.io/docs/instrumentation/erlang/

Helpful libraries and instrumentation libraries for JavaScripts projects like [express](http://expressjs.com/) and [Koa](https://koajs.com/)